docs/app.md

```diff
@@ -4,29 +4,34 @@ An app installed in an organization from a repository.
-| `name`            | `String`                           | Name of the app, from the repository. E.g. `"Tokens"`.                                                                   |
-| `registryAddress` | `String`                           | Address of the aragonPM registry for this app.                                                                           |
-| `registry`        | `String`                           | Name of the aragonPM registry for this app. E.g. `"aragonpm.eth"`                                                        |
-| `sourceUrl`       | `String`                           | URL of the app source code.                                                                                              |
-| `version`         | `String`                           | The current version of the app.                                                                                          |
-| `kernelAddress`   | `String`                           | The address of the kernel.                                                                                               |
-| `isForwarder`     | `Boolean`                          | Whether the app can act as a forwarder.                                                                                  |
-| `tags`            | `String[]`                         | Tags associated with the app.                                                                                            |
+| Name                | Type                               | Description                                                                                                              |
+| ------------------- | ---------------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `abi`               | `Abi`                              | The Ethereum contract ABI of the app contract.                                                                           |
+| `address`           | `String`                           | The address of the app proxy contract (never changes).                                                                   |
+| `appId`             | `String`                           | The appName but encoded.                                                                                                 |
+| `appName`           | `String`                           | The app ENS identifier. E.g. `"token-manager.aragonpm.eth"`                                                              |
+| `author`            | `String`                           | App author, from the repository. E.g. `"Aragon Association"`.                                                            |
+| `chainId`           | `String`                           | Chain ID for this app.                                                                                                   |
```

Comment:
I think this `chainId` value is a bit confusing. Because it represent other things on different contexts (e.g. ethers.js). If I'm not mistaken what we meant here would be the AragonEnvrionment name of this app.

Reply:
Agree to remove it.